docs/src/extend/custom-processors.md

```diff
@@ -102,15 +105,15 @@ See [Specify a Processor](../use/configure/plugins#specify-a-processor) in the P
 
 ## File Extension-named Processor
 
-If a processor name starts with `.`, ESLint handles the processor as a **file extension-named processor** especially and applies the processor to the kind of files automatically. People don't need to specify the file extension-named processors in their config files.
+If a custom processor name starts with `.`, ESLint handles the processor as a **file extension-named processor**. ESLint applies the processor to files with that filename extension automatically. Users don't need to specify the file extension-named processors in their config files.
```

Comment:
To reviewer: I have trouble understanding this line.

Is it saying that as long as the file extension named processor is included in the plugin, it runs? and therefore users don't need to specify the file extension-named processors in their config files?

would appreciate clarification so i can adjust phrasing accordingly 😄

Reply:
Originally, processors were enabled automatically when a plugin containing them was loaded. These auto-enabled processors were named according to file extension, such as:

```js
module.exports = {
    processors: {
        ".md": { /* processor */ }
    }
};
```

So, if the name begins with a dot, we assume it should be auto-enabled.

The way we want people to name processors today is with a descriptive name, like so:

```js
module.exports = {
    processors: {
        "markdown": { /* processor */ }
    }
};
```

Because this name doesn't begin with a dot, we know it shouldn't be applied automatically.